docs/api/1.x/filtering.rst

```diff
@@ -56,6 +56,33 @@ Sub-objects
 
 * ``/collection?field.subfield=value``
 
+Search in array fields
+----------------------
+
+* ``/collection?contains_field=value``
+
+  Matches any records whose ``field`` array field contains ``value``. Values can
+  be an integer, a string, a list of integers or a list of strings. In case of
+  values are a list, only matches records whose field contains all the values
+  listed.
```

Comment:
What happens if ``field`` is not an array?

This can also match objects, but I guess it doesn't matter because almost any object contains you could consider doing is better expressed as a bunch of equality filters?

Reply:
I guess we should raise a 400 for objects here.